draft-irtf-cfrg-hpke.md

```diff
@@ -279,7 +279,11 @@ operations, roles, and behaviors of HPKE:
 
 - `(skX, pkX)`: A KEM key pair used in role X; `skX` is the private
   key and `pkX` is the public key.
-- `pk(skX)`: The public key corresponding to private key `skX`.
+- `pk(skX)`: The KEM public key corresponding to the KEM private key
+  `skX`. Depending on its use and the KEM, this is either the computation
+  of the public key using the private key, or just syntax expressing the
+  retrieval of the public key assuming it is stored along with the private
+  key object.
```

Comment:
This seems like an implementation detail, so I'd remove it.

Reply:
I added this because #84 mentioned that for example RSA does not have this clear of a relation between public key and secret key; at least computing the public key from the secret key involves a lot of computation in RSA. We could mention it elsewhere in the document? Maybe just move the new text to the Future KEMs section?

Reply:
Moved to Future KEMs for now.

Reply:
👍